packages/emotion/src/__tests__/withStyle.test.tsx

```diff
@@ -136,6 +138,28 @@ describe('@withStyle', async () => {
     }
   }
 
+  class WrapperComponent extends React.Component {
+    render() {
+      return (
+        <div>
+          <ThemeableComponent />
+        </div>
+      )
+    }
+  }
+
+  it.only('can be found and tested with ReactTestUtils', async () => {
+    const rootNode = document.createElement('div')
+    document.body.appendChild(rootNode)
+
+    // eslint-disable-next-line react/no-render-return-value
+    const rendered = ReactDOM.render(<WrapperComponent />, rootNode)
+    ReactTestUtils.findRenderedComponentWithType(
+      rendered as any,
+      (ThemeableComponent as any).originalType
+    )
```

Comment:
This test has no assertion. Or will it throw an error if its not found?

Reply:
it will and then the test fails